### PR TITLE
Swagger open api documentation feature

### PR DESCRIPTION
**Pull request by anovosiolovGD**

### CODE REVIEW

**Comment by ppanasencoGD** [comment]
please change target branch to develop

File: src/main/java/com/nvs/store/config/SecurityConfig.java [hunk 1]
```diff
@@ -17,22 +15,37 @@ public class SecurityConfig {
     private final JwtAuthFilter jwtAuthFilter;
     private final AuthenticationProvider authenticationProvider;
 
+//    @Bean
```
Comment (DrVa97) [inline src/main/java/com/nvs/store/config/SecurityConfig.java:18]: Perhaps the commented code could be removed.

File: src/main/java/com/nvs/store/service/ProductService.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    public ResponseEntity<Product> updateProduct() {
+        return null;
+    }
+
+    public void deleteProduct(Long id) {
+        productRepository.deleteById(id);
+    }
+
+
```
Comment (DrVa97) [inline src/main/java/com/nvs/store/service/ProductService.java]: Unnecessary spaces.

File: pom.xml [hunk 3]
```diff
@@ -72,17 +71,31 @@
 			<scope>runtime</scope>
 			<optional>true</optional>
 		</dependency>
+        <dependency>
```
Comment (acostasgd) [inline pom.xml:74]: Please format the code.

File: src/main/resources/open-api-documentation/openapi.json [hunk 4]
```diff
@@ -0,0 +1,306 @@
+{
```
Comment (acostasgd) [inline src/main/resources/open-api-documentation/openapi.json:1]: The resource file for swagger may to be maintained, it is possible to use swagger annotations to prepare API documentation?

Comment (anovosiolovGD) [reply src/main/resources/open-api-documentation/openapi.json:1]: At the beginning I was using Annotations but founded it more complicate than using just Swagger editor and export it into JSON file 
Then we can use this JSON in Swagger editor or Postman

**Review by DrVa97 — APPROVED** [review]

**Review by acostasgd — APPROVED** [review]

**Pull request closed without merging**